bin/inx-chronicle/src/stardust_inx/mod.rs

```diff
@@ -192,3 +193,36 @@ impl HandleEvent<Report<Syncer>> for InxWorker {
         Ok(())
     }
 }
+
+#[async_trait]
+impl HandleEvent<IsHealthy> for InxWorker {
+    async fn handle_event(
+        &mut self,
+        _cx: &mut ActorContext<Self>,
+        IsHealthy(sender): IsHealthy,
+        inx_client: &mut Self::State,
+    ) -> Result<(), Self::Error> {
+        let node_status = NodeStatus::try_from(inx_client.read_node_status(NoParams {}).await?.into_inner())
+            .map_err(InxError::InxTypeConversion)?;
+
+        let mut healthy = true;
+        healthy &= node_status.is_healthy;
+
+        let latest_inserted_ms = self.db.get_latest_milestone().await?;
+        healthy &= latest_inserted_ms.map_or(false, |latest_inserted_ms| {
+            node_status.confirmed_milestone.milestone_info.milestone_index == latest_inserted_ms
+        });
+
+        let first_ms = node_status.tangle_pruning_index + 1;
```

Comment:
I'm not sure at the moment, is the pruning index still included by the node, or the latest index that has been pruned away?

Reply:
the latter (the above calculated index points to the first prunable milestone)